fuzzing/libfuzzer/common/config/python_packages.yaml

```diff
@@ -0,0 +1,15 @@
+---
+# Need to enclose few of the below dependencies within quotes as the cmd string
+# will be formed with spaces and pip install will fail without the quotes.
+Default:
+ asv>=0.5.1
+ "meson >=0.56, <0.57"
+ "tomli>=1.1.0"
+ "tomli-w>=0.4.0"
+ openpyxl
+
+"18.04":
```

Comment:
Like I had mentioned in my previous review, you don't need to support 18.04, you can remove this section.